scripts/lintTs.mjs

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as os from 'os';
+import * as path from 'path';
+import * as url from 'url';
+import chalk from 'chalk';
+import {ESLint} from 'eslint';
+import pLimit from 'p-limit';
+import {getPackagesWithTsConfig} from './buildUtils.mjs';
+
+// we want to limit the number of processes we spawn
+const cpus = Math.max(1, os.cpus().length - 1);
+const mutex = pLimit(cpus);
+
+const fix = process.argv.slice(2).some(arg => arg === '--fix');
+
+const monorepoRoot = path.resolve(url.fileURLToPath(import.meta.url), '../..');
+
+// TODO: remove this list at some point and run against all packages
+const packagesToTest = [
+  'babel-jest',
+  'babel-plugin-jest-hoist',
+  'diff-sequences',
+  'jest-source-map',
```

Comment:
Only four packages? That’s not su bad. It is indeed good idea to have this added!

Reply:
this is a list of which we lint, not the ones excluded. running on e.g. `expect` gives almost 400 errors, so I stopped there 😅

Reply:
Ah.. So that’s a good start then ;D